cmd/medius/images/push.go

```diff
@@ -261,8 +320,22 @@ func (b *buildAndPublish) pushImage(containerDisk v1.Image, name string) error {
 	return nil
 }
 
+func (b *buildAndPublish) pushImageIndex(containerDiskIndex v1.ImageIndex, name string) error {
+	if !b.Options.DryRun {
```

Comment:
This is nicer:
```
if !b.Options.DryRun {
  b.Log.Infof("Dry run enabled, not pushing %s image index", name)
  return nil
}
b.Log.Infof("Pushing %s image index", name)
if err := b.Repo.PushImageIndex(b.Ctx, containerDiskIndex, name); err != nil {
	b.Log.WithError(err).Error("Failed to push image image")
	return err
}
```

Reply:
I agree, but I wasn't responsible for the code style of it, I just made it similar to `pushImage` code style.